### PR TITLE
do only delete affected processed files on compress command

### PR DESCRIPTION
**Pull request by achimfritz**
if you have a lot of images and want to compress them with the CLI Command do not remove all processed files on each call, but only the ones compressed.

### CODE REVIEW

File: Classes/Command/CompressImagesCommandController.php [hunk 1]
```diff
@@ -53,12 +53,13 @@ public function compressCommand()
         /** @var FileStorage $fileStorage */
         foreach ($this->fileStorageRepository->findAll() as $fileStorage) {
             $excludeFolders = GeneralUtility::trimExplode(',', (string)$settings['exludeFolders'], true);
-            $files = $this->fileRepository->findAllNonCompressedInStorageWithLimit($fileStorage, 100, $excludeFolders);
+            $files = $this->fileRepository->findAllNonCompressedInStorageWithLimit($fileStorage, 1, $excludeFolders);
```
Comment (schmitzal) [inline Classes/Command/CompressImagesCommandController.php]: Why did you change the limit to 1 file? It should stay at 100 or even better be configurable.

File: composer.json [hunk 2]
```diff
@@ -15,6 +15,11 @@
             "name": "Alessandro Schmitz",
             "role": "Developer",
             "homepage": "https://www.interlutions.de/"
+        },
```
Comment (schmitzal) [inline composer.json]: Please do not mention you as an author.
I will mention you and your company in the CHANGELOG.md

File: Classes/Command/CompressImagesCommandController.php (hunk 1 above)
Comment (achimfritz) [reply Classes/Command/CompressImagesCommandController.php]: argh, sure, this was a testing typo
